### PR TITLE
DOCSP-44950 TOC Relabel

### PR DESCRIPTION
**Pull request by lindseymoore**
Jira: https://jira.mongodb.org/browse/DOCSP-44950
Staging: https://deploy-preview-83--docs-ruby.netlify.app/

### CODE REVIEW

File: source/getting-started.txt [hunk 1]
```diff
@@ -6,12 +6,12 @@ Getting Started
...
   :titlesonly:
 
-  installation
-  reference/driver-compatibility
-  support
+  Installation <installation>
+  Driver Compatibility <reference/driver-compatibility>
+  Support <support>
+
+This section describes how to install the driver, installation prerequisites
```
Comment (mongoKart) [inline source/getting-started.txt:16]: [oxford comma](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#commas)

```suggestion
This section describes how to install the driver, installation prerequisites,
```

File: source/index.txt [hunk 2]
```diff
@@ -8,6 +8,21 @@ Ruby MongoDB Driver
...
+
+    Get Started <getting-started>
+    Tutorials <tutorials>
+    Connection & Configuration <reference/connection-and-configuration>
+    Working with Data <reference/working-with-data>
+    Schema Operations <reference/schema-operations>
+    API <https://mongodb.com/docs/ruby-driver/current/api/>
+    Release Notes <release-notes>
+    Additional Resources <reference/additional-resources>
+    Contribute to the Driver <contribute>
```
Comment (mongoKart) [inline source/index.txt:24]: Could shorten

```suggestion
    Contribute <contribute>
```

File: source/reference/working-with-data.txt [hunk 3]
```diff
@@ -12,16 +12,16 @@ implements for inserting, updating and retrieving data from MongoDB.
...
-  /reference/gridfs
-  /reference/change-streams
-  /reference/sessions
-  /reference/transactions
-  /reference/in-use-encryption
+  CRUD </reference/crud-operations>
+  Bulk Writes </reference/bulk-operations>
+  Projection </reference/projection>
+  Aggregation </reference/aggregation>
+  Map-Reduce </reference/map-reduce>
```
Comment (mongoKart) [inline source/reference/working-with-data.txt:19]: MapReduce seems conventional

```suggestion
  MapReduce </reference/map-reduce>
```

**Review by mongoKart — APPROVED** [review]
lgtm w/ small changes

**Review by sarahemlin — CHANGES_REQUESTED** [review]
LGTM with Mike's suggestions

**Pull request merged**